src/python/grpcio/grpc/aio/_interceptor.py

```diff
@@ -503,6 +505,20 @@ async def _proxy_writes_as_request_iterator(self):
                 break
             yield value
 
+    async def _write_to_iterator_queue_interruptible(
+            self, request: RequestType, call: InterceptedCall):
+        # Write the specified 'request' to the request iterator queue using the
+        # specified 'call' to allow for interruption of the write in the case
+        # of abrupt termination of the call.
+        if self._status_code_task is None:
+            self._status_code_task = self._loop.create_task(call.code())
+
+        _, _ = await asyncio.wait(
```

Comment:
nit: Since we are not using the returned coroutines, I would recommend to ignore them.